docs/docs/welcome.mdx

```diff
@@ -0,0 +1,90 @@
+---
+slug: /
+title: Welcome to Nebari's documentation
+---
+
+export const invisible_table = {
+   color: 'blue',
+   lineHeight: 10,
+   padding: '1.5em',
+   background: 'white',
+   backgroundColor: 'white',
+}
+
+:::warning We are currently undergoing a rename from [QHub](https://docs.qhub.dev/) to Nebari.   
+:construction: Please excuse our dust as we build out our new docs :) :construction:
+:::
+
+[INSERT NEBARI LOGO HERE]
```

Comment:
Since we will not have the logo before the sprint let's remove this as this looks unfinished